nearuplib/nodelib.py

```diff
@@ -294,7 +294,7 @@ def restart_nearup(net,
     logging.info("Nearup has been restarted...")
 
 
-def stop_native():
+def stop_native(timeout=None):
```

Comment:
Why not assign timeout to DEFAULT_WAIT_TIMEOUT instead of checking if None below

Reply:
Makes sense! fixed